docs/api/ref/Metadata.md

```diff
@@ -279,6 +279,9 @@ interface ImplementationMetadata {
      *  `fdc3.getCurrentChannel` and `fdc3.leaveCurrentChannel` are implemented by
      *  the Desktop Agent.*/
     "UserChannelMembershipAPIs": boolean;
```

Comment:
Not related to this PR, but it's weird that some of these properties are in quotes and some aren't.

Reply:
Yes it is, thats probably my bad from when this was first added - they're not needed.

Reply:
This was resolved in another PR and this one rebased on top.